TLM/TLM/Manager/Impl/TrafficLightManager.cs

```diff
@@ -32,17 +34,17 @@ public class TrafficLightManager : AbstractCustomManager, ICustomDataManager<Lis
 
 		public bool SetTrafficLight(ushort nodeId, bool flag, ref NetNode node, out ToggleTrafficLightUnableReason reason) {
 #if DEBUGTTL
```

Comment:
Looks like old define, change to `#if DEBUG`? Multiple occurrences in file.

Reply:
I'll do this last before review. There are some `#if` which guard logging code but no debug switch check, so that's a lot of possible log spam is hiding there.